kobo/apps/mfa/models.py

```diff
@@ -9,6 +10,27 @@
 )
 
 
+class KoboMFAPerUserActivation(models.Model):
+
+    class Meta:
+        verbose_name = 'Per-user activation'
+        verbose_name_plural = 'Per-user activations'
+    user = models.ForeignKey('auth.User', on_delete=models.CASCADE)
+
+    def __str__(self):
+        # Used to display the username in Django Admin instead of a representation
+        # of `KoboMFAPerUserActivation` object.
+        # It could be done with `list_display = ('user',)` in
+        # `KoboMFAPerUserActivationAdmin`
+        return self.user.username
```

Comment:
What about getting rid of it and use `list_display` as I said in the comment.  It will show exactly the same thing and ~~`KoboMFAPerUserActivation`~~  `MfaAvailableToUser` object string representation will not be altered.

Reply:
no thanks :) please see comment above